vars/publishToLogServer.groovy

```diff
@@ -1,20 +1,12 @@
-def showUrl(Map logServer, String destDir) {
-    def httpRootLocation = '/var/www/logs/'
-    if (destDir.contains(httpRootLocation)) {
-        echo ('logs published to http://' + logServer.addr + destDir.replace(httpRootLocation, '/'))
-    }
-}
-
-def call(Map logServer, String srcDir, String destDir, boolean mayThrow = true) {
+def call(String authority, String srcDir, String destDir, boolean mayThrow = true) {
```

Comment:
If `srcDir` would be removed (according to previous comment), then `mayThrow` can also be removed.